### ISSUE TITLE
arcane crafting NEI Bug 1.1.8

### ISSUE THREAD

**Issue by Nightarix** [jadedcat/AS2Configs#15]
not sure if this is with the plugin or caused by something in the pack, but if you check the recipe for the witching gadgets "spinning wheel" the arcane crafting portion of NEI will get stuck on it and won't show any other recipe.

edit: come to think of it, might just be cause i hadn't started with Thaumcraft yet which could be confusing NEI, will test. 
edit2: nope, used cheat sheet to test. still happens in the order of 1. look at shaped arcane recipe for spinning wheel via NEI 2. look at any other recipe.

**Comment by pjfranke** [comment]
I am unable to duplicate this issue on 1.1.8, and no one else has noted they are experiencing the issue.

**Comment by vScourge** [comment]
I can reproduce this consistently in 1.1.8, actually.  It's as he described, and only starts happening when you view an arcane crafting recipe.  Once it starts it stays stuck on that recipe as the first page in NEI (scrolling shows the correct ones still) until you restart MC.

**Comment by MarshMilo100** [comment]
can you give a copy of your log? vscourge

**Comment by vScourge** [comment]
Sure, which log exactly?

**Comment by MarshMilo100** [comment]
nevermind. can you unlock the pack and try to manually update thaumic nei or thaumcraft nei plugin, and see if the problem persists?

**Comment by MarshMilo100** [comment]
which arcane crafting recipe page is buggin out? thaumic nei or thaumcraft nei plugin?

**Comment by MarshMilo100** [comment]
can you give any sort of screenshot of what the problem is, I am still not quite clear on which gui is messing things up.

**Comment by MarshMilo100** [comment]
I am also not able to reproduce this bug

**Comment by vScourge** [comment]
I did some mod shuffling/testing and narrowed this down to the "gud_thaumicnei" mod.  When AS2 went to 1.1.8, it updated that mod from 1.2.3 to 1.3.0.  If I revert my SMP server and client back to 1.2.3 the problem goes away.  There's many NEI-related mods in AS2 so maybe it's some interaction between them, I just know rolling this one back fixed it.  I haven't checked for/tested any releases newer than 1.3.0

For the record, to repro this I just hit "R" on the golem animation core to see the recipe.  That requires arcane crafting, and once shown any later view of a recipe shows the anim core one on the first page.  I'm usually inside the arcane crafting terminal GUI from Thaumic Energistics when I do this, but I don't know if that's relevant.

**Comment by MarshMilo100** [comment]
Thank you for the input on the version downgrade.

 I don't understand what you mean in the second part of your post.

**Comment by pjfranke** [comment]
Thanks for you update, with that I was able to test what's going on, and narrow down the issue.

It seems to be not tied to the Spinning Wheel, but actually maybe the Thaumcraft "Table", which is used in the Spinning wheel recipe.  If you look up the recipe for the Deconstruction Table, it has the same issue.

I have reported this to the mod author on their mod's page comment section, as they don't seem to have a bugtracker available.  For now, we'll downgrade the version to 1.2.3 to fix the issue.
